x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/scalar/datetime/DayOfMonth.java

```diff
@@ -6,43 +6,38 @@
 package org.elasticsearch.xpack.sql.expression.function.scalar.datetime;
```

Comment:
briefly all other sql date functions are no var args, changes were made because no longer inherit from unaryscalarfunction. Nothing else really changed.